hack/make-pjs.sh

```
@@ -0,0 +1,191 @@
+            | jq -r '.zz_generated_metadata
+                  | if has("variant") then .variant else "" end')
+
+  names=$(yq -ojson "${RELEASE_REPO}/${ci_operator_config}" \
+          | jq -jc \
+              '[
+                  .tests[]
+                  | select(has("restrict_network_access") and (.restrict_network_access|not))
+                  | .as
+              ]')
+
+  yq -ojson "${RELEASE_REPO}"/ci-operator/jobs/${ORG}/${REPO}/${ORG}-${REPO}-${BRANCH}-*.yaml \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)